### PR TITLE
B 2008 put not validated form item to center

### PR DESCRIPTION
**Pull request by MohammadPCh**
#2008

### CODE REVIEW

File: src/components/views/CreateMilestone.jsx [hunk 1]
```diff
@@ -34,6 +34,7 @@ function CreateMilestone(props) {
   const { id: campaignId, slug: campaignSlug } = props.match.params;
 
   const campaign = useCampaign(campaignId, campaignSlug);
+  const [form] = Form.useForm();
```
Comment (aminlatifi) [inline src/components/views/CreateMilestone.jsx:37]: What's the usage of this?

Comment (aminlatifi) [reply src/components/views/CreateMilestone.jsx:37]: I mean form is not used anywhere just passing to <Form> ... </Form> component

**Comment by aminlatifi** [comment]
@RamRamez Please test it

**Comment by RamRamez** [comment]
> @RamRamez Please test it

In Payment and Bounty forms, view is not scrolling to the Description section when criteria is not met.

**Comment by aminlatifi** [comment]
> > @RamRamez Please test it
> 
> In Payment and Bounty forms, view is not scrolling to the Description section when criteria is not met.

@RamRamez It's probably related to setting `id` field. Please fix it yourself

**Comment by RamRamez** [comment]
> > > @RamRamez Please test it
> > 
> > 
> > In Payment and Bounty forms, view is not scrolling to the Description section when criteria is not met.
> 
> @RamRamez It's probably related to setting `id` field. Please fix it yourself

It's done in PR #2015

**Comment by aminlatifi** [comment]
Changes applied under #2015

**Pull request closed without merging**